test/iterator.js

```diff
@@ -432,3 +439,8 @@ metatests.testSync('Iterator.join with prefix and suffix', test => {
   const actual = iter(array).join(', ', '[', ']');
   test.strictSame(actual, '[1, 2, 3, 4]');
 });
+
+metatests.testSync('Iterator.join on empty iterator', test => {
```

Comment:
I'd prefer to use `test` for consistency.

Reply:
All of the other tests on `Iterator.join` use `testSync()`, so it is consistent right now.